terraform/executor/Dockerfile-0.11

```diff
@@ -12,7 +12,7 @@ ARG HELM_VERSION=2.16.9
 ARG TFSWITCH_VERSION=0.8.832
 ARG TF_HELM_VERSIONS="0.6.0 0.5.1 0.5.0 0.4.0 0.3.2 0.3.1 0.3.0 0.2.0 0.1.0"
 ARG TF_ACME_VERSIONS="1.0.0 0.6.0 0.5.0 0.4.0 0.3.0"
-ARG TF_AIVEN_VERSIONS="1.0.19 1.0.20 1.1.0 1.1.1 1.1.2 1.1.3 1.1.4"
+ARG TF_AIVEN_VERSIONS="1.0.19 1.0.20 1.1.0 1.1.1 1.1.2 1.1.3 1.1.4 1.3.5 2.0.6"
```

Comment:
latest v1 because people couldn't have dependended on the unavailable version

latest v2 for same reason, and because they need 0.13 to download via tf